docs/analysis-workflows/dna-sanger-wgs-vc.md

```diff
@@ -0,0 +1,47 @@
+---
+id: dna-sanger-wgs-vc
+title: Sanger WGS Variant Calling
+sidebar_label: Sanger WGS Variant Calling
+platform_key: DOCS_DNA_PIPELINE
+---
+
+Whole genome sequencing (WGS) aligned CRAM files are processed through the Sanger WGS Variant Calling Workflow as tumour/normal pairs. The ARGO DNA Seq pipeline has adopted the [Sanger Whole Genome Sequencing Analysis Docker Image](https://quay.io/wtsicgp/dockstore-cgpwgs:2.1.0) as the base workflow. For details, please see the latest version of the [ARGO Sanger WGS Variant Calling workflow](https://github.com/icgc-argo/sanger-wgs-variant-calling/releases).
```

Comment:
Can update to use new github repo: https://github.com/icgc-argo-workflows/sanger-wgs-variant-calling/releases